README.md

```diff
@@ -1,122 +1,129 @@
-# eap-microprofile-test-suite
- a small standalone test suite for MicroProfile on WF/EAP
+# EAP Microprofile test suite
+ 
+A small standalone test suite for MicroProfile on WildFly/EAP.
+
+## Supported MicroProfile version
+
+Tests have been executed against WildFly and EAP XP versions supporting 
+[MicroProfile 4.1 specs](https://projects.eclipse.org/projects/technology.microprofile/releases/4.1) 
+
+## Supported MicroProfile specs
+
+Currently, the following MicroProfile specs are supported:
+
+* [MicroProfile Config](https://github.com/eclipse/microprofile-config)
+* [MicroProfile JWT RBAC](https://github.com/eclipse/microprofile-jwt-auth)
+* [MicroProfile Health](https://github.com/eclipse/microprofile-health)
+* [MicroProfile Fault Tolerance](https://github.com/eclipse/microprofile-fault-tolerance)
+* [MicroProfile Metrics](https://github.com/eclipse/microprofile-metrics)
+* [MicroProfile OpenAPI](https://github.com/eclipse/microprofile-open-api)
+* [MicroProfile OpenTracing](https://github.com/eclipse/microprofile-opentracing)
+
+More tests could be added, specifically covering the MicroProfile 
+[REST Client](https://github.com/eclipse/microprofile-rest-client), 
+[Reactive Messaging](https://github.com/eclipse/microprofile-reactive-messaging) and
+[Reactive Streams Operators](https://github.com/eclipse/microprofile-reactive-messaging) (just tech preview at the 
+moment) specs.
 
 ## Requirements for the testsuite
- - JDK 8+
- - Docker
+- JDK 8+
+- Docker
  
- Maven is not a hard requirement as `mvnw` is delivered with this repository.
-
- Podman with docker command bridge can be used as a Docker alternative or you can use a system property `-Ddocker.command=podman`.
+Maven is not a hard requirement as `mvnw` is delivered with this repository.
+
+Podman can be aliased to be used as a Docker alternative - i.e. `alias docker=podman`, or you can pass the 
+`-Ddocker.command=podman` system property to the test suite when executing Maven.
+
+## Modules
+
+### MicroProfile specs testing modules
+- [microprofile-config](./microprofile-config) - contains MicroProfile Config deployments and tests
+- [microprofile-fault-tolerance](./microprofile-fault-tolerance) - contains MicroProfile Config deployments and tests
+- [microprofile-health](./microprofile-health) - contains MicroProfile Config deployments and tests
+- [microprofile-jwt](./microprofile-jwt) - contains MicroProfile Config deployments and tests
+- [microprofile-metrics](./microprofile-metrics) - contains MicroProfile Config deployments and tests
+- [microprofile-open-api](./microprofile-open-api) - contains MicroProfile Config deployments and tests
+- [microprofile-opentracing](./microprofile-opentracing) - contains MicroProfile Config deployments and tests
```

Comment:
It seems there is copy paste error in above lines - `...MicroProfile Config...`

Reply:
+1, I removed the section as per @marekkopecky [comment](https://github.com/jboss-eap-qe/eap-microprofile-test-suite/pull/248#discussion_r859612846)